clang/test/Modules/pr121245.cpp

```diff
@@ -0,0 +1,90 @@
+// RUN: rm -rf %t
```

Comment:
Please add a comment that the test results in `llvm_unreachable` in debug builds so debug build should be used to investigate problems with the test.